server/src/main/java/com/coincoinche/websockets/messages/GameStart.java

```diff
@@ -0,0 +1,7 @@
+package com.coincoinche.websockets.messages;
+
+public class GameStart extends SocketMessage {
+  public GameStart() {
+    super(MessageType.GAME_START);
+  }
+}
```

Comment:
Why having different classes that initialize a different message instead of just using `MessageType.GAME_START`, etc. directly?

Reply:
I was assuming we were gonna put more fields in here but indeed for now I guess I can simplify